lib/Backends/CPU/libjit/libjit.cpp

```diff
@@ -1392,4 +1392,123 @@ libjit_dump_tensor(uint8_t *tensor, size_t *tensorDim, size_t numDimsTensor,
     break;
   }
 }
+
+void find_min_max_f(float *tensor, size_t size, float &min, float &max) {
+  for (size_t i = 1; i < size; ++i) {
```

Comment:
nit: I don't like how this function assumes min/max have already been set to `tensor[0]`. Can you do it here at the start of the function?